lib/src/boollist.dart

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:collection' show ListMixin;
+import 'dart:typed_data' show Uint32List;
+
+import 'unmodifiable_wrappers.dart' show NonGrowableListMixin;
+
+/// A BoolList implementation for Dart to store boolean values.
```

Comment:
```dart
/// A space-efficient list of boolean values.
```

Being nit-picky here :) 
The "for Dart" is redundant in a Dart file. Repeating the class name isn't saying anything new. The first line of a DartDoc is intended to grab the attention of someone looking for the functionality that the class is providing, without going into too much detail.